### PR TITLE
CREATE_PROJECT: Consistent name for zlib

### PR DESCRIPTION
**Pull request by henke37**
The feature name didn't match with other code.

### CODE REVIEW

File: devtools/create_project/codeblocks.cpp [hunk 1]
```diff
@@ -58,8 +58,11 @@ StringList getFeatureLibraries(const BuildSetup &setup) {
 	for (FeatureList::const_iterator i = setup.features.begin(); i != setup.features.end(); ++i) {
 		if (i->enable && i->library) {
 			std::string libname;
-			if (!std::strcmp(i->name, "libz") || !std::strcmp(i->name, "libcurl")) {
+			if (!std::strcmp(i->name, "libcurl")) {
 				libname = i->name;
+			} else if (!std::strcmp(i->name, "zlib")) {
+				libname = "libz";
+				libraries.push_back("libvorbisfile");
```
Comment (Die4Ever) [inline devtools/create_project/codeblocks.cpp:65]: looks like libvorbisfile shouldn't be here

**Comment by Die4Ever** [comment]
you're probably gonna need to change some `configure.engine` files, right?

File: devtools/create_project/codeblocks.cpp (hunk 1 above)
Comment (henke37) [reply devtools/create_project/codeblocks.cpp:65]: Whopsie.

**Comment by henke37** [comment]
> you're probably gonna need to change some `configure.engine` files, right?

I don't think I will need to, since they already build correctly on platforms that don't use `create_project`.

**Comment by bluegr** [comment]
Thanks, the naming of the zlib library is more consistent with this. Merging

**Pull request merged**